birdhouse/components/jupyterhub/jupyterhub_config.py.template

```diff
@@ -136,6 +136,9 @@ if os.environ['WORKSPACE_DIR'] != jupyterhub_data_dir:
 
 container_gdrive_settings_path = join(container_home_dir, ".jupyter/lab/user-settings/@jupyterlab/google-drive/drive.jupyterlab-settings")
 host_gdrive_settings_path = os.environ['JUPYTER_GOOGLE_DRIVE_SETTINGS']
+resource_limits = {tuple(lim[:2]): dict(li.split("=") for li in lim[2:] if "=" in li)
```

Comment:
So the format looks like
```
resource_limits = {
  'user:user1': {'mem_limit': '10G},
  'group:group1': {'cpu_limit': '3', 'mem_limit': '3G'},
}
```
Can you add this expected format as a comment to ease future reader?